package-lock.json

```
@@ -1,17 +1,18 @@
 {
   "name": "vscode-lizard",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "vscode-lizard",
-      "version": "1.0.1",
+      "version": "1.0.2",
```

```suggestion
      "version": "1.0.1",
```

VS Code's publication tool will update this when I publish a new release.